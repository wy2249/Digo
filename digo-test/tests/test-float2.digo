func digo_main() void {
  var a, b, c float
  a = 3.14159267
  b = -2.71828
  c = a + b
  println("%f", c)
}
