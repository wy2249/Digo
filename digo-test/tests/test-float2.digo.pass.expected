0.42331267